doc/os_api/endpoints/pure/security_endpoint.md

```diff
@@ -32,16 +32,44 @@ Security endpoint (13)
 
 ```json
 {
-  "endpoint":13,
-  "status":204,
-  "uuid":123
+  "endpoint": 13,
+  "status": 204,
+  "uuid": 123
 }
 ```
 
 Parameters:
- - *status* - 423 when the EULA is not accepted or battery level is critical otherwise 204 when phone is unlocked or 403 when phone is locked
+
+- *status* - 204 when phone is unlocked or 403 when phone is locked
+
+**Response Payload Structure**
+
+```json
+{
+  "endpoint": 13,
+  "body": {
+    "reason": "2"
+  },
+  "status": 423,
+  "uuid": 123
+}
+```
+
+Parameters:
+
+- *status* - 423 when phone is unlocked or 403 when phone is locked
+- *reason* - block reason, see code snippets below
+
+```c++
+    enum class BlockReason
+    {
+        EulaNotAccepted = 2,
+        BatteryCriticalLevel
+    };
+```
```

Comment:
shouldn't we mention that those status codes are HTTP codes? If it's already written down then just resolve the comment :)

Reply:
It is written here:
https://github.com/mudita/MuditaOS/blob/ff31f28e047a0e55878e091fe08c05854a42a6ac/doc/os_api/protocol_description/common_parameters_explanation.md

> status - HTTP-like status code